src/app/lib/navbar/navbar.component.scss

```diff
@@ -0,0 +1,119 @@
+@import '../colors.scss';
+
+$navbar-color: $charcoal-blue !default;
+$navbar-brand: $blue !default;
+$navbar-height: 53px !default;
+$navbar-app-padding: 0 75px 0 25px !default;
+$navbar-text: $white !default;
+$navbar-text-inactive: $gray-300 !default;
+$navbar-z-index: 1030 !default;
+
+.navbar.fixed-top::ng-deep {
```

Comment:
It's my understanding that `ng-deep` [is deprecated. ](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep). However, I'm not sure of a different way to handle this.